tests/Yaapii.Atoms.Tests/IO/ZipTest.cs

```diff
@@ -66,15 +68,78 @@ public void HasEntry()
                 newFile = File.Create(Path.Combine(folder, "FileToZipThree.txt"));
                 newFile.Close();
 
-                var streamOfZipped = new Zip(folder);
+                var streamOfZipped = new Zip(folder).Stream();
 
-                var archive = new ZipArchive(streamOfZipped.Stream());
-                Assert.True(archive.GetEntry(Path.Combine(folder, "FileToZipTwo.txt")) != null);
+                var archive = new ZipArchive(streamOfZipped);
+                Assert.True(archive.GetEntry("FileToZipTwo.txt") != null);
             }
             finally
             {
                 Directory.Delete(folder, true);
             }
+        }   
+
+        [Fact]
+        public void ZipSomeFiles()
+        {
+            var folder = new TempDirectory("abc\\data").Value();
```

Comment:
please always use the TempDirectory class inside a using-block, so it is removed after:
```csharp
using(var root= new TempDirectory("abc\\data"))
{
   var folder = root.Value();
   ...
}
```